tests/functional/specs/control_tags/choice.cy.ts

```diff
@@ -3,77 +3,40 @@ import {
   choicesConfig,
   choicesMultipleSelectionConfig, choicesSelectLayoutConfig,
   simpleData
-} from 'data/control_tags/choice';
-import { FF_DEV_2007 } from '@heartexlabs/ls-test/feature-flags';
+} from '../../data/control_tags/choice';
```

Comment:
Is it accountable change?

Reply:
I think it's a bit early to introduce aliases without proper transition. Something was complaining about the path, like TS or something. And they won't work in monorepo right now.